src/js/electron/windows-pre-25.ts

```diff
@@ -0,0 +1,33 @@
+import {app, dialog} from "electron"
+import fs from "fs-extra"
+import os from "os"
+import {join} from "path"
+
+const VAR = "LocalAppData"
+
+function getDir() {
+  if (VAR in process.env) {
+    const dir = process.env[VAR]
+    return join(dir, "Brim")
+  } else {
+    return join(app.getPath("home"), "AppData", "Local", "Brim")
+  }
+}
+
+function getExe() {
+  return join(getDir(), "Brim.exe")
+}
+
+export function windowsPre25Exists() {
```

Comment:
Would you mind copying a bit of your PR description into the code as a comment for posterity?

Reply:
Yes, thank you.